### PR TITLE
adding note in application section about HPC

### PR DESCRIPTION
**Pull request by ChristinaLK**
This would be one way to resolve the conversation threads about Singularity + HPC centers.

### CODE REVIEW

File: _episodes/07-docker-image-examples.md [hunk 1]
```diff
@@ -28,13 +28,36 @@ In this [GitHub Actions example](../_extras/e01-github-actions), you can learn m
...
+There isn't yet a working example of how to use Docker container images on a shared
+computing system, partially because each system is slightly different, but the
+following resources show what it can look like:
+
+- Singularity lesson
+- HPC Carpentry
+- Pawsey?
+
+## Seeking Examples
+
```
Comment (sstevens2) [inline _episodes/07-docker-image-examples.md:60]: 👍

File: _episodes/07-docker-image-examples.md [hunk 2]
```diff
@@ -28,13 +28,36 @@ In this [GitHub Actions example](../_extras/e01-github-actions), you can learn m
...
 
-<!--- Placeholder for 
+<!--- Placeholder for
 ## Geospatial Example
 
 Ask @mkuzak to make a PR to add extra for <https://github.com/escience-academy/docker-gdal-demo>
 
 -->
 
+## On an HPC Cluster
```
Comment (dme26) [inline _episodes/07-docker-image-examples.md:38]: Maybe add words to make this heading self-contained?

File: _episodes/07-docker-image-examples.md (hunk 1 above)
Comment (dme26) [reply _episodes/07-docker-image-examples.md:60]: This is an excellent addition.

File: _episodes/07-docker-image-examples.md (hunk 2 above)
Comment (ChristinaLK) [reply _episodes/07-docker-image-examples.md:38]: can you give an example of what you mean?

File: _episodes/07-docker-image-examples.md [hunk 3]
```diff
@@ -28,13 +28,36 @@ In this [GitHub Actions example](../_extras/e01-github-actions), you can learn m
...
+so often people create their container as a Docker container image, so they can
+run it using Docker or Singularity.
+
+There isn't yet a working example of how to use Docker container images on a shared
+computing system, partially because each system is slightly different, but the
+following resources show what it can look like:
+
+- Singularity lesson
+- HPC Carpentry
+- Pawsey?
```
Comment (sstevens2) [inline _episodes/07-docker-image-examples.md:57]:
```suggestion
- [Singularity lesson](https://carpentries-incubator.github.io/singularity-introduction/)
- [HPC Carpentry](https://www.hpc-carpentry.org/)
- [Pawsey](https://pawseysc.github.io/singularity-containers/)
```

**Review by sstevens2 — APPROVED** [review]
Added links to the resources section.  Otherwise I think this looks like a good start.

File: _episodes/07-docker-image-examples.md (hunk 3 above)
Comment (ChristinaLK) [reply _episodes/07-docker-image-examples.md]: I had already made this change and forgotten to push. 🤦‍♀️

File: _episodes/07-docker-image-examples.md [hunk 4]
```diff
@@ -52,9 +52,10 @@ There isn't yet a working example of how to use Docker container images on a sha
 computing system, partially because each system is slightly different, but the
 following resources show what it can look like:
 
-- Singularity lesson
-- HPC Carpentry
-- Pawsey?
+- [Introduction to Singularity](https://carpentries-incubator.github.io/singularity-introduction/): See the episode titled "
+Running MPI parallel jobs using Singularity containers
+"
```
Comment (sstevens2) [inline _episodes/07-docker-image-examples.md:57]:
```suggestion
Running MPI parallel jobs using Singularity containers"
```

**Review by sstevens2 — APPROVED** [review]
Looks good to me.  minor suggestion to reunite the quotation with its statement in the md but it will get there in the rendered one version in anycase.

**Pull request merged**